README.md

```diff
@@ -367,44 +367,46 @@ Special builds:
 | 1.x-centos7 | dockerfile/centos_7 |
 | 2.x-centos7 | dockerfile/centos_7 |
 
-## How to push changes (for maintainers)
+## Release policy
 
-When the change is about specific tarantool version or versions
-range, update all relevant fixed versions & rolling versions
-according to the pipelines listed above.
+All images are pushed to [Docker Hub](docker_hub_tags).
 
-When the change is about the environment at all, all versions
-need to be updated.
+Fixed version tags (`x.y.z`) are frozen: we never update them.
 
-Add a new release (say, x.y.z). Create / update rolling
-versions x and x.y in master, create fixed version x.y.z.
+Rolling versions are updated to the last fixed version tags:
 
-A maintainer is responsible to check updated images.
+- `x.y` == `x.y.<last-z>` (`==` means 'points to the same image')
+- `1` == `1.<last-y>.<last-z>`
+- `2` == `2.<last-y>.<last-z>`
+- `latest` == `2`
 
-[1]: https://tarantool.io/en/doc/1.9/dev_guide/release_management/#how-to-make-a-minor-release
+Special builds (CentOS) are updated with the same policy as rolling versions:
 
-## How to push images (for maintainers)
-Gitlab-CI jobs after the images builds push it to its local
-images repository in the format:
-    registry.gitlab.com/tarantool/docker:<tag>
-To push collected images into the 'docker.io' repository with
-the following format:
-    docker.io/tarantool/tarantool:<tag>
-the following scripts can be used:
+- `1.x-centos7` image offers a last `1.<last-y>.<last-z>` release
+- `2.x-centos7` image offers a last `2.<last-y>.<last-z>` release
 
-```bash
-echo List of available tags: \
-    `grep " VER: " .gitlab-ci.yml | awk -F"'" '{print $2}'`
-```
+[docker_hub_tags]: https://hub.docker.com/r/tarantool/tarantool/tags
 
-```bash
-for tag in <list of tags> ; do \
-    echo "============= $tag ==============" ; \
-    docker pull registry.gitlab.com/tarantool/docker:$tag && \
-    docker tag registry.gitlab.com/tarantool/docker:$tag \
-        tarantool/tarantool:$tag && \
-    docker push tarantool/tarantool:$tag ; \
-    echo "$tag push resulted with: $?" ; \
-done
-```
+### Exceptional cases
+
+As an exception we can deliver an important update for the existing tarantool
+release within `x.y.z-r1`, `x.y.z-r2`, ... tags.
```

Comment:
Uh, I see you've mentioned them here. But still believe it worth to mention that updates are the same kind of fixed tags. But, well, I'm not insisting (because current description is clear enough)

Reply:
I agree, it'll be more strict. I'll try to reword it as you proposed and will check how it looks.